packages/synapse-react-client/src/lib/containers/entity_finder/EntityFinder.tsx

```diff
@@ -208,71 +213,85 @@ export const EntityFinder: React.FunctionComponent<EntityFinderProps> = ({
 
   return (
     <SynapseErrorBoundary>
-      <div className="bootstrap-4-backport EntityFinder">
-        <div className="EntityFinder__Search" data-active={searchActive}>
+      <div className="EntityFinder">
+        <Box
+          display="flex"
+          justifyContent="flex-end"
+          alignItems="center"
+          pb={1}
+        >
           <>
-            {searchActive ? (
+            {!searchActive ? (
```

Comment:
Switched the order of the buttons in this section, so rather than always rendering the Search bar and hiding it using `aria-hidden` when not searching for files, instead render either the "Search for Files" button or the "Back to Browse" button and Search bar.

Reply:
Files view -- 

main: 
<img width="1488" alt="main_EntityFinderModal_Files" src="https://github.com/Sage-Bionetworks/synapse-web-monorepo/assets/26949006/0cfb98ce-93d5-4420-b7dc-9286747c24f2">

feature:
<img width="1465" alt="feature_EntityFinderModal_Files" src="https://github.com/Sage-Bionetworks/synapse-web-monorepo/assets/26949006/dccd99ec-ca27-4e9f-a9e8-f48aed873b37">

Reply:
Search view -- 

main:
<img width="1484" alt="main_EntityFinderModal_Search" src="https://github.com/Sage-Bionetworks/synapse-web-monorepo/assets/26949006/79e57849-6b9f-4e84-866e-951d65e0feb5">

feature: 
<img width="1465" alt="feature_EntityFinderModal_Search" src="https://github.com/Sage-Bionetworks/synapse-web-monorepo/assets/26949006/acec6b24-3a7d-4962-8ca0-92a91eaaf60b">

Reply:
Animation wasn't preserved and the table isn't aligned with the search field in my experience: 

main:

https://github.com/Sage-Bionetworks/synapse-web-monorepo/assets/17580037/79dfa59d-fd84-4eed-acb9-30b55d0a36c1

feature:

https://github.com/Sage-Bionetworks/synapse-web-monorepo/assets/17580037/70519fb7-19e6-473b-8cbb-ac4045dd7b99

Reply:
Oh good call! I've brought back the transition and correct margin in the next commit